### PR TITLE
feat(reactions): Add 👋 and 🔥 call reactions

### LINKED ISSUE #13343: Add wave emoji to call reactions
:wave:

### PR DESCRIPTION
**Pull request by nickvergessen**
### ☑️ Resolves

* Fix #13343 

### 🏁 Checklist

- [x] ⛑️ Tests (unit and/or integration) are included or not possible
- [x] 📘 API documentation in `docs/` has been updated or is not required
- [x] 🔖 Capability is added or not needed

### CODE REVIEW

File: src/components/CallView/shared/ReactionToaster.vue [hunk 1]
```diff
@@ -46,8 +46,13 @@ const reactions = {
 	'❤️': 'Heart.gif',
 	'🎉': 'Party.gif',
 	'👏': 'Clap.gif',
+	'👋': 'Wave.gif',
 	'👍': 'Thumbs-up.gif',
 	'👎': 'Thumbs-down.gif',
+<<<<<<< Updated upstream
+=======
+	'🔥': 'Fire.gif',
+>>>>>>> Stashed changes
```
Comment (SystemKeeper) [inline src/components/CallView/shared/ReactionToaster.vue:55]: No.

**Review by SystemKeeper — CHANGES_REQUESTED** [review]

File: src/components/CallView/shared/ReactionToaster.vue [hunk 2]
```diff
@@ -46,8 +46,13 @@ const reactions = {
 	'❤️': 'Heart.gif',
 	'🎉': 'Party.gif',
 	'👏': 'Clap.gif',
+	'👋': 'Wave.gif',
 	'👍': 'Thumbs-up.gif',
 	'👎': 'Thumbs-down.gif',
+<<<<<<< Updated upstream
```
Comment (Antreesy) [inline src/components/CallView/shared/ReactionToaster.vue:52]: 👋

**Review by Antreesy — APPROVED** [review]
Tested

**Comment by Antreesy** [comment]
Question arise: how it's supposed to work in federated calls with older versions?

**Pull request merged**